tasks.schema.json

```diff
@@ -0,0 +1,334 @@
+          },
+          "properties": {
+            "type": "object",
+            "properties": {
+              "EditableOptions": {
+                "type": "string",
+                "enum": [
+                  "True",
+                  "False"
+                ]
+              }
+            }
+          },
+          "options": {
+            "type": "object",
+            "additionalProperties": true
+          }
+        }
+      }
+    },
+    "dataSourceBindings": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "target": {
+            "type": "string"
+          },
+          "endpointId": {
+            "type": "string"
+          },
+          "dataSourceName": {
+            "type": "string"
+          },
+          "parameters": {
+            "type": "object"
+          },
+          "resultTemplate": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "sourceDefinitions": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "target": {
+            "type": "string"
+          },
+          "endpoint": {
+            "type": "string"
+          },
+          "selector": {
+            "type": "string"
+          },
+          "keySelector": {
+            "type": "string"
+          },
+          "authKey": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "execution": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Execution options for this task",
+      "properties": {
+        "Node": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "Bash": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "AzurePowerShell": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "PowerShell": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "PowerShell3": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "PowerShellExe": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "Process": {
+          "$ref": "#/definitions/executionObject"
+        },
+        "RM:ManualIntervention": {
+          "$ref": "#/definitions/executionObject"
+        }
+      }
+    },
+    "messages": {
+      "type": "object"
+    },
+    "$schema": {
+      "type": "string"
+    }
+  },
+  "definitions": {
+    "executionObject": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "target": {
+          "type": "string",
+          "description": "The target file to be executed. You can use variables here in brackets e.g. $(currentDirectory)\filename.ps1"
+        },
+        "argumentFormat": {

```

Comment:
@bryanmacfarlane Is there a description for argumentFormat? I've searched but have not found what argument format is suppose to do. For most examples is just an empty string.  Fox example: Can I pass arguments here?

Reply:
It is from legacy handlers and it should not be used.